wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -319,7 +319,7 @@ type ResetFailedPtxs =
         "For all transactions in CPtxWontApply condition, \
         \reset them to CPtxApplying condition so that they will \
         \be passed to resubmition"
-    :> WRes Get ()
+    :> WRes Get NoContent
```

Comment:
In theory this should be already in `master`, but as long as it doesn't create conflict, I don't mind which variant we need 😉